h/authentication.py

```diff
@@ -21,6 +21,13 @@
 from .oauth import IClientFactory
 
 
+def normalize_authorization_header(request):  # bw compat
+    """Convert any X-Annotator-Auth-Token header to an Authorization header."""
+    token = request.headers.pop('X-Annotator-Auth-Token', None)
```

Comment:
I don't care that much either way but my thought was that this helped ensure that no other code anywhere in the back-end mentioned or depended on this header anymore.